src/components/SoundBox/ColumnOfBoxes/SimpleBox/SimpleBox.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from "react";
+import styled, { css } from "styled-components";
+
+export const Button = styled.button`
+  background: darkgray;
+  border: 0;
+  height: 125px;
+  width: 125px;
+  display: block;
+  margin-top: 2px;
+  margin-bottom: 2px;
+
+  ${props =>
+    props.clicked && props.color === "red" && css`background: #e2818a;`};
+  ${props =>
+    props.clicked && props.color === "violet" && css`background: #a8a2e8;`};
+  ${props =>
+    props.clicked && props.color === "green" && css`background: #98e1d6;`};
+  ${props =>
+    props.clicked && props.color === "sand" && css`background: #f0cc90;`};
```

Comment:
What do you think if instead of on general Button with so many props we create custom color button components? And extend from the common one. It will look like ```ViolerButton``` and ```PurpleButton``` and then you will just use the proper buttons for the appropriate columns. It will increase readability and avoid future support concerns.